### PR TITLE
Create 240607-deprecation-adjustment-transaction-type

### PR DESCRIPTION
**Pull request by Amy-Roberts**
# Description

* Include a summary of the new content you're adding or the reason for the change. 
* Include relevant context.
* Do not link to work items.
* Any PNG images should be compressed using [Tiny PNG](https://tinypng.com/) or equivalent.

## Type of change

Please delete options that are not relevant.

- [ ] New document(s)/updating existing
- [ ] Fixes
- [ ] Styling
- [ ] Bug fix (non-breaking change which fixes an issue)

## Reviews and merging

You are responsible for getting your PR merged. Address review comments promptly and **make sure to merge the PR when ready**.
Feel free to 'Enable automerge' - your PR will automatically merge when accepted and passing the build.

### CODE REVIEW

File: blog/240607-deprecation-adjustment-transaction-type [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: "2024-10-10: Deprecation of `Adjustment in/out` transaction types for Sync for Expenses"
+date: "2024-06-07"
+tags: ["Deprecation"]
+draft: true
```
Comment (pzaichkina) [inline blog/240607-deprecation-adjustment-transaction-type:5]: should this be in draft, or are you ready to publish this?

File: blog/240607-deprecation-adjustment-transaction-type [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+title: "2024-10-10: Deprecation of `Adjustment in/out` transaction types for Sync for Expenses"
+date: "2024-06-07"
```
Comment (pzaichkina) [inline blog/240607-deprecation-adjustment-transaction-type:3]: pls amend

File: blog/240607-deprecation-adjustment-transaction-type [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+draft: true
+authors: Amy-Roberts
+---
+On **October 10, 2024**, we will remove the `Adjustment in` and `Adjustment out` transaction types from the Sync for Expenses [Create expense transaction](/sync-for-expenses-api#/operations/create-expense-transaction) endpoint. 
+
+<!--truncate-->
+
+After the change, you will need to use the [Create adjustment transaction](/sync-for-expenses-api#/operations/create-adjustment-transaction) endpoint to represent an adjustment when syncing expenses. 
+
+This endpoint is currently only available for our QuickBooks Desktop integration.
```
Comment (pzaichkina) [inline blog/240607-deprecation-adjustment-transaction-type:14]: is QBD right? the [list of supported tr types](https://docs.codat.io/expenses/overview#supported-integrations) seems to indicate Xero and QBO are the ones with adjustments, so are we expecting the new endpoint to cover those?

File: blog/240607-deprecation-adjustment-transaction-type (hunk 1 above)
Comment (Amy-Roberts) [reply blog/240607-deprecation-adjustment-transaction-type]: Removed

File: blog/240607-deprecation-adjustment-transaction-type (hunk 3 above)
Comment (Amy-Roberts) [reply blog/240607-deprecation-adjustment-transaction-type:13]: Right now we won't add Xero as no one is using it or has requested it

File: blog/240607-deprecation-adjustment-transaction-type (hunk 2 above)
Comment (Amy-Roberts) [reply blog/240607-deprecation-adjustment-transaction-type:3]: What needs changing? The date for today? @pzaichkina

**Review by pzaichkina — APPROVED** [review]

**Pull request merged**